wrangler-core/src/main/java/io/cdap/directives/parser/ParseExcel.java

```diff
@@ -165,8 +167,8 @@ public List<Row> execute(List<Row> records, final ExecutorContext context)
 
                   case NUMERIC:
                     if (HSSFDateUtil.isCellDateFormatted(cell)) {
-                      newRow.add(name, cell.getDateCellValue());
-                      value = cell.getDateCellValue().toString();
+                      newRow.add(name, formatter.formatCellValue(cell));
```

Comment:
Is there any test case that need to be updated to capture this scenario?

Reply:
Done, added a test